README.md

```diff
@@ -1,21 +1,46 @@
 # Foreman Ansible Modules [![Build Status](https://travis-ci.org/theforeman/foreman-ansible-modules.svg?branch=master)](https://travis-ci.org/theforeman/foreman-ansible-modules)
 
-This repository contains Ansible modules for interacting with a Foreman server API and various plugin APIs such as Katello.
+This repository contains Ansible modules for interacting with the Foreman API and various plugin APIs such as Katello.
 
 ## Goals
 
 The intent of this repository is to be a place that community members can develop or contribute modules. The goals of this repository are:
 
   * centralized location for community modules
   * a single repository to clone for interacting with Foreman & plugins
-  * an intermediate landing place for modules before pushing them to Ansible community
-  * repository maintainers will be working to push the modules into Ansible proper [https://github.com/ansible/ansible/tree/devel/lib/ansible/modules/remote_management/foreman](https://github.com/ansible/ansible/tree/devel/lib/ansible/modules/remote_management/foreman)
+  * source for the official Ansible collection (TBD)
 
 ## Branches
 
 * `master` - current development branch, using both `nailgun` and `apypie` libraries
 * `nailgun` - the state of the repository before the switch to the `apypie` library started, `nailgun` is the only dependency
 
+## Supported Foreman and plugins versions
+
+### `apypie` based modules
+
+Modules that use the `apypie` library should support any currently stable Foreman release.
+
+We actively test them against the latest stable Foreman release and the matching set of plugins.
+
+### `nailgun` based modules
```

Comment:
Perhaps this should mention we're phasing this out and link to https://github.com/theforeman/foreman-ansible-modules/issues/274

Reply:
I've added that info in the "branches" section above